draft-ietf-quic-transport.md

```diff
@@ -1444,9 +1444,20 @@ language from Section 3 of {{!I-D.ietf-tls-tls13}}.
       TransportParameter parameters<22..2^16-1>;
    } TransportParameters;
 
+   struct {
+     uint8[4] addr;
+   } IPv4addr;
+
+   struct {
+     uint8[16] addr;
+   } IPv6addr;
```

Comment:
You can use a typedef for this:

```
typedef uint8[16] IPv6addr;
```

Reply:
There are no typedefs in the [TLS 1.3 draft](https://tools.ietf.org/html/draft-ietf-tls-tls13-28).

Reply:
The way you say this in TLS is:

```
uint8 IPv6addr[16];
```

Reply:
Ah yes, this is much simpler, thanks.  Done in c2647e37ea6af229d201b542bb2a8f372f872440.